.verb.md

```diff
@@ -1,3 +1,9 @@
+## Quick Start 
+Assuming you want to add a TOC to README.md:
+1. ```$ npm install -g markdown-toc```
+2. Edit README.md and insert the following line where you want the TOC inserted:<br />```<-- toc -->```
```

Comment:
This should be 

```<!-- toc -->```

Missing `!`

Reply:
Oops, good catch!   I'll fix the PR in a moment ...

Reply:
No problem. Your quick start was very useful to me as a new user 👍